core/templates/base-components/base-content.component.css

```diff
@@ -0,0 +1,82 @@
+/*
+  NOTE to developers: Only the '.css' files should be edited. The styling should
+  be generalized enough that the autogenerated '.rtl.css' files should work
+  without any manual input.
+  This can be generated by running 'python -m scripts.generate_rtl_css' script.
```

Comment:
The '.rtl.css' files can be generated by ...

(it's not clear what "this" is).

Reply:
Done